### ISSUE TITLE
does not run last test from batch

### ISSUE THREAD

**Issue by mutt13y** [magnusbaeck/logstash-filter-verifier#91]
When have a file with multiple inputs the last test is not run
say I have 20 tests the run will end with

Comparing message 19 of 20 test.yml

if I adjust the last test to make it fail no error is printed either I assume it did not run

**Comment by magnusbaeck** [comment]
Could you please provide a minimal example file that illustrates the problem?

**Comment by mutt13y** [comment]
I managed to fix the issue. by adding the proper yaml markers (--- to the start and ... to the end)
It seems that without these it sometimes does not execute the last test.

If you still need an example I can try to extract a minimal example from my config.

**Comment by magnusbaeck** [comment]
Yes, could you please do that? I haven't experienced any problems with testcases being dropped.
